### PR TITLE
GE Server Installer fails to create link to manual under htdocs

### PR DESCRIPTION
**Pull request by tst-awankhede**
fix for  #188
permission changes from other requests. 
duplicate code lines removed
cutter permission issue seen in the log, so modified it to the parent.

### CODE REVIEW

**Comment by tst-awankhede** [comment]
I signed it!

File: earth_enterprise/src/installer/common.sh [hunk 1]
```diff
@@ -250,6 +250,10 @@ create_links()
         ln -s $BASEINSTALLDIR_VAR/run $BASEINSTALLDIR_OPT/run
     fi
 
+    if [ ! -L "$BASEINSTALLDIR_OPT/share/doc" ]; then
```
Comment (tst-lsavoie) [inline earth_enterprise/src/installer/common.sh]: I'm a little confused by this - should we be checking if $BASEINSTALLDIR_OPT/gehttpd/htdocs/shared_assets/docs exists?

Comment (tst-awankhede) [inline earth_enterprise/src/installer/common.sh]: yeah. that's the thing I was talking about this am. the wrong check I left in there . sending one that will fix it.

Comment (tst-awankhede) [inline earth_enterprise/src/installer/common.sh]: please review , its tested and the check is corrected to verify "if the link does not exists, then"

**Pull request merged**